helm/values.yaml

```diff
@@ -35,16 +107,15 @@ resources: {}
   #   cpu: 100m
   #   memory: 128Mi
 
-nodeSelector: {}
+podAnnotations: {}
+
+nodeSelector:
+  beta.kubernetes.io/os: linux
 
 tolerations: []
 
 affinity: {}
 
-node:
```

Comment:
Let's keep this, continuing my comment above, since there is a controller component now we are going to need to separate node values from controller values.

ref:
https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/values.yaml#L86